quilt/tools/command.py

```diff
@@ -489,9 +489,9 @@ def _print_children(children, prefix, path):
     def _print_node(node, prefix, child_prefix, name, path):
         name_prefix = u"─ "
         if isinstance(node, dict):
-            node_type = NodeType(node.pop(TYPE_KEY))
+            node_type = NodeType(node["type"])
```

Comment:
Shouldn't we still use constants for these? That way the linter will catch errors if we make typos.